cumulusci/tasks/metadata_etl/tests/test_layouts.py

```diff
@@ -177,3 +180,331 @@ def test_skips_existing_related_list(self):
+<platformActionList>
+    <actionListContext>{action_list_context}</actionListContext>
+    {optional_first_action_item}
+    <platformActionListItems>
+        <actionName>Edit</actionName>
+        <actionType>StandardButton</actionType>
+        <sortOrder>{}</sortOrder>
+    </platformActionListItems>
+    <platformActionListItems>
+        <actionName>FeedItem.TextPost</actionName>
+        <actionType>QuickAction</actionType>
+        <sortOrder>{}</sortOrder>
+    </platformActionListItems>
+    {optional_last_action_item}
+</platformActionList>
+"""
+# Empty action item and empty action list
+EMPTY_ACTION_ITEM = """
+    <platformActionListItems>
+        <actionName>{action_name}</actionName>
+        <actionType>{action_type}</actionType>
+        <sortOrder>{expected_order}</sortOrder>
+    </platformActionListItems>
+"""
+EMPTY_ACTION_LIST = """
+<platformActionList>
+    <actionListContext>{action_list_context}</actionListContext>
+    {optional_action_item}
+</platformActionList>
+"""
+
+
+class TestAddRecordPlatformActionListItem:
+    def test_adds_action_item_to_existing_list_place_last(self):
+        # options scenario:
+        #   adding Quick Action to layout with existing action item list
+        #   not setting place_first, so should be last in action list
+        options = {
+            "action_type": "QuickAction",
+            "action_name": "pkg__mockObject.TestQuickAction",
+        }
+        task = create_task(AddRecordPlatformActionListItem, options)
+        assert (
+            task._place_first is False
+        ), "task._place_first should be false when not setting `place_first` task_option"
+
+        # Mocks: build our existing action list and create our metadata tree
+        #   "Record" context
+        #   default sort order (0, 1)
+        #   no optional
+        mock_action_list = MOCK_EXISTING_ACTION_LIST.format(
+            0,
+            1,
+            action_list_context="Record",
+            optional_first_action_item="",
+            optional_last_action_item="",
+        )
+        metadata = metadata_tree.fromstring(
+            MOCK_EMPTY_LAYOUT.format(action_list_scenario=mock_action_list).encode(
+                "utf-8"
+            )
+        )
+        mock_action_list_size = len(
+            metadata._get_child("platformActionList").findall("platformActionListItems")
+        )
+
+        # Creating expected action item/list xml and metadata
+        #   Expected context = "Record"
+        #   The action_list_items <sortOrder> (positional *args) can be dynamically set from the mock_action_list_size or range
+        #   using our optional_last_action_item to set our expected_action_item placement
+        expected_action_item = EMPTY_ACTION_ITEM.format(
+            expected_order=mock_action_list_size, **options
+        )
+        expected_action_list = MOCK_EXISTING_ACTION_LIST.format(
+            *tuple([n for n in range(mock_action_list_size)]),
```

Comment:
`range` is also a sequence, so you can unpack it here:
```suggestion
            *range(1, mock_action_list_size + 1),
```